pulsar-client-cpp/python/src/config.cc

```diff
@@ -142,5 +142,7 @@ void export_config() {
             .def("receiver_queue_size", &ReaderConfiguration::setReceiverQueueSize)
             .def("reader_name", &ReaderConfiguration::getReaderName, return_value_policy<copy_const_reference>())
             .def("reader_name", &ReaderConfiguration::setReaderName)
+            .def("reader_name", &ReaderConfiguration::getSubscriptionRolePrefix, return_value_policy<copy_const_reference>())
+            .def("reader_name", &ReaderConfiguration::setSubscriptionRolePrefix)
```

Comment:
These 2 lines should have the `"subscription_role_prefix"` and not `"reader_name"`

Reply:
Sorry, fixed.